packages/stores/src/queries-external/github/get-file.ts

```diff
@@ -0,0 +1,68 @@
+import { KVStore } from "@keplr-wallet/common";
+import { HasMapStore } from "@keplr-wallet/stores";
+import { makeObservable } from "mobx";
+
+import { GITHUB_RAW_DEFAULT_BASEURL } from "..";
+import { ObservableQueryExternalBase } from "../base";
+
+export class ObservableQueryFileInner<
+  T
+> extends ObservableQueryExternalBase<T> {
```

Comment:
Does this function need to be in the store package? As I understand it, part of the purpose of putting some queries in a separate stores package is because we will someday ideally have docs so that other devs can use our open source queries. Does this one fit that model? Or am I misunderstanding something? Perhaps @jonator can shed some light as well.

Reply:
I agree that this seems to be tightly coupled to our app. I'll move this to the `web` package.